### PR TITLE
feat: CommentComposer component

### PR DESCRIPTION
**Pull request by wereHamster**
To show these components in the catalog I need a `t()` function. To make it convenient I created a a `docs.t.js` file in the component folder and expose that function to the whole catalog page using imports. But perhaps it would be better to store those strings in JSON files so that they can be used to seed the spreadsheet you have. And it would also allow us to show the required translation keys in the catalog pages.

image

### CODE REVIEW

File: src/components/CommentComposer/CommentComposerHeader.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import {css} from 'glamor'
+import {MdCheck, MdEdit} from 'react-icons/lib/md'
+import colors from '../../theme/colors'
+import {sansSerifMedium16, sansSerifRegular14} from '../Typography/styles'
+
+const styles = {
+  root: css({
+    display: 'flex',
+    alignItems: 'center',
+    background: colors.whiteSmoke,
```
Comment (uxengine) [inline src/components/CommentComposer/CommentComposerHeader.js]: Just for the record:
As discussed, these colors need to replaced by functional names once the color scheme is better defined:

File: src/components/CommentComposer/docs.t.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default k => {
```
Comment (uxengine) [inline src/components/CommentComposer/docs.t.js]: Let's just follow the form component model and use component attributes like "placeholder" and "replyLabel", which will make the whole t() mocking obsolete.

Comment (wereHamster) [reply src/components/CommentComposer/docs.t.js]: This doesn't scale well. If you compose a few of these components into larger ones, you'll end up having to pass dozens of props to the top-level component, and it increases the likelihood of a name conflict. Already now we have two components which would require a `replyLabel`: `CommentActions` and `CommentComposer`.

Comment (uxengine) [reply src/components/CommentComposer/docs.t.js]: Good point about composition, I agree. So let's stick to your current version and defer further improvements of the translations setup to later.

**Pull request merged**